actions/translation/loadDeckTranslations.js

```diff
@@ -10,8 +10,9 @@ export default function loadDeckTranslation(context, payload, done) {
             context.executeAction(serviceUnavailable, payload, done);//TODO improve
             return;
         } else {
-            if (res && res[0] && res[0].language) {
-                res = res.reduce((a, c) => {a.push(c.language); return a;}, []);
+            if (res && res.length[0] && res[0].language) {
```

Comment:
res && res.length > 0 && res[0].language
is correct, right?

Reply:
yes this is a mistake, but currently I don't use this action, don't know how that code made it in the commit :)